### DISCUSSION TITLE
USB-C SMT component out of stock.  Alternatives?

### DISCUSSION THREAD

**Discussion by kevin-s** [alvarop/usb_c_cable_tester#17] [Q&A]
The USB-C connector is currently out of stock.  Is there an easy way to search for a equal sized and installed replacement?  Is it common for parts to come in and out of stock from time to time, so I should check again next week?  

![12-25-2023 8-49-22 PM](https://github.com/alvarop/usb_c_cable_tester/assets/2939475/7da5e4a2-76f7-4fd9-9202-cdd865ccbffd)

Thank you so much for the detailed writeup on ordering my own boards.  I appreciate the info and guidance!!

**Comment by alvarop** [comment]
I'll have to take a look. I doubt alternates will have the same footprint.
I'm not sure how often they restock to be honest!

Hopefully I can take a look over the next few days and maybe do another release with a more available one

**Reply by alvarop** [reply]
I couldn't quickly find a "drop-in" replacement. This one _looks_ close, but the mechanical mounting holes don't quite line up. https://jlcpcb.com/partdetail/ShouHan-TYPE_C_24P_QCHT143/C5156604

I'd have to do another board spin, unfortunately :(

**Comment by alainm** [comment]
Thanks a lot.
I'll be waiting, along with many friends that will get the tester from me
:)  :)
